### PR TITLE
Updating slack blame to 0.5 format

### LINKED ISSUE #4: allow storing a github -> slack user mapping
the majority of my team doesn't have matching emails in github and slack, so it would be nice if we could manually map those.

ideally this would be in the secrets somehow, could just be a simple `github_user=slack_user,github_user2=slack_user2` list which would be easy-ish to put in secrets and expand in the plugin

### LINKED ISSUE #5: allow variables in messages
might be nice to expand `$$VARIABLE` style variables in messages, that would allow for super customizable alerts that show as much or as little information as an end user wants

### PR DESCRIPTION
**Pull request by donny-dont**
- Updated to 0.5 format.
- Added mapping - Fixes #4 
- Added templating - Fixes #5

### CODE REVIEW

File: DOCS.md [hunk 1]
```diff
@@ -14,45 +11,84 @@ designated channel. This lets developers get feedback on their own feature
...
-  * **image_attachments** - a list of image attachments to append (only one will
-    be randomly selected per build)
-* **failure** - the options for a build failure
-  * **icon** - an emoji or image url for the bot
-  * **username** - the username for a failed build (defaults to "drone")
-  * **message** - the message for a failed build
-  * **image_attachments** - a list of image attachments to append (only one will
-    be randomly selected per build)
+```bash
+drone secret add --image=npm \
```
Comment (tboerger) [inline DOCS.md]: Wrong image

File: DOCS.md [hunk 2]
```diff
@@ -14,45 +11,84 @@ designated channel. This lets developers get feedback on their own feature
...
+    octocat/hello-world SLACK_TOKEN xxxxxxx
+```
+
+Then sign the YAML file after all secrets are added.
```
Comment (tboerger) [inline DOCS.md:52]: I would reference to the regular docs instead

Comment (donny-dont) [inline DOCS.md:52]: @bradrydzewski said to do that cause people don't read the docs.

**Comment by tboerger** [comment]
Maybe you can apply the same changes like https://github.com/drone-plugins/drone-slack/pull/1?

**Comment by appleboy** [comment]
When time to merge this PR? 👍 Nice Improvement.

**Comment by tboerger** [comment]
LGTM

**Comment by kzaitsev** [comment]
LGTM

**Pull request merged**